dydx3/modules/private.py

```diff
@@ -1131,11 +1131,33 @@ def get_trading_rewards(
             },
         )
 
+    def get_liquidity_provider_rewards_v2(
```

Comment:
We could call it `get_lp_rewards`?

Reply:
i like v2 better, `get_lp_rewards` gets a bit confusing